### PR TITLE
Adds a Storybook page for the GrowlManager

### PR DESCRIPTION
**Pull request by scures**
### Summary
Fixes https://github.com/rancher/dashboard/issues/7450

Adds the information on how/where to use the GrowlManager component.

### CODE REVIEW

**Comment by kwwii** [comment]
Here's my take on the informational text:


# GrowlManager

GrowlManager component is used to display asynchronous responses of specific actions as a overlay. Details shown in each messge include the severity, a summary, and any details.

### When to use it

* To provide real-time information to the User about out-of-view events or background processes.
* To display a message which requires User action.
* To display non-critical information that won’t disrupt a workflow.


### When not to use it
* For error messages related to specific form fields.
* For messages that are related to a specific component or view (use inline alerts instead).

### Usage

Messages with different levels of severity can be displayed in the GrowlManager component by dispatching actions to the `growl` store.


#### Info (default)
For events of lower relative importance, or for generic use.

```ts
this.$store.dispatch('growl/info', {
    title:   'Info',
    message: 'This is an info message',
  }, { root: true });
```

#### Success
For reporting the successful completion of an operation or action.

```ts
this.$store.dispatch('growl/success', {
    title:   'Success',
    message: 'This is an success message',
  }, { root: true });
```

#### Warning
For events that require user attention.
```ts
this.$store.dispatch('growl/warning', {
    title:   'Warning',
    message: 'This is an warning message',
  }, { root: true });
```

#### Error
When things go wrong.

For events that require user attention.
```ts
this.$store.dispatch('growl/error', {
    title:   'Error',
    message: 'This is an error message',
  }, { root: true });
```


<Canvas>
  <Story name="GrowlManager">
    {Template.bind({})}
  </Story>
</Canvas>

### Import

<Source
  language='js'
  light
  format={false}
  code={`
     import GrowlManager from '@shell/components/GrowlManager.vue'
  `}
/>

**Review by cnotv — CHANGES_REQUESTED** [review]
Overall good job there, just some hints about the imports for the store.

Some comments for the future: we should decouple it from the store, so it may be exported as component, by adding props and emitter, then bind to the store in the layout. 
The only concern there is code duplication.

File: .storybook/preview.js [hunk 1]
```diff
@@ -14,25 +14,71 @@ require('../shell/plugins/i18n');
...
-      if ( msg?.includes('{')) {
+      if (msg?.includes('{')) {
         const formatter = new IntlMessageFormat(msg, state.selected);
         return formatter.format(args);
       }
 
       return msg;
     }
+  },
+  modules: {
```
[the remark this answers is not in the document]
Comment (cnotv) [reply .storybook/preview.js:35]: I tried this way and it seems working, plus it should not apply to all the components:

```ts
export const Template = () => ({
  components: { GrowlManager },
  store: { ...store }.registerModule('growl', growl),
  template:   '<GrowlManager />',
})
```

It should help to keep changes within the story and easy to find, without extend the configuration everywhere.

Note for the readers (@rak-phillip ): to create a store within the component is currently not possible, as it requires `i18n`

[the remark this answers is not in the document]
Comment (cnotv) [reply .storybook/preview.js:35]: In any case, I'll create an issue to remove the state from a basic component like this one, as this should be handled in the view.

https://github.com/rancher/dashboard/issues/7561

**Review by cnotv — APPROVED** [review]
LGTM

**Pull request merged**